app/styleguide/page.tsx

```diff
@@ -1425,17 +1426,20 @@ function Page() {
 
             <Example>
               <TypeAheadWithOther
-                label="Example input"
+                label="Institution"
+                fieldName="institution"
                 graphqlQuery={AffiliationsDocument}
-                helpText="Help text describing what types of data the user can search for"
                 setOtherField={setOtherField}
+                required={true}
```

Comment:
When adding TypeAheadWithOther field into the User Profile page, I found that I needed some additional props.